docs/sphinx/source/changelog/v2.0.0b0.rst

```diff
@@ -0,0 +1,93 @@
+  Generally, when working with raw power data it should be converted to
+  right-labeled energy with :py:func:`~rdtools.normalization.energy_from_power`
+  before being used with these normalization functions (:pull:`105`, :pull:`108`).
+* Remove ``low_power_cutoff`` parameter in :py:func:`~rdtools.filtering.clip_filter` (:issue:`84`).
+* Rename `soiling.srr_analysis` to :py:class:`~rdtools.soiling.SRRAnalysis` (:pull:`168`).
+* Double the default value of the ``max_timedelta`` in :py:func:`~rdtools.normalization.interpolate`
+  and :py:func:`~rdtools.normalization.interpolate_series` to be twice the
+  median timedelta (:pull:`182`).
+* Support varying logic in soiling module for defining cleaning events from shifts and
+  precipitation with ``clean_criterion`` parameter. Behavior of ``clean_criterion='precip_and_shift'``
+  has changed relative to prior versions using ``precip_clean_only=True`` (:pull:`176`).
+* Remove ``random_seed`` parameter from soiling module. Functionality can be obtained by running
+  ``numpy.random.seed()`` outside of RdTools functions. (:pull:`176`)
+* Many kwargs have changed name (but not input order) to bring nomenclature into 
+  closer alignment with the `DuraMAT pv-terms project <https://duramat.github.io/pv-terms>`_: (:pull:`185`)
+
+  * :py:func:`~rdtools.aggregation.aggregation_insol` first kwarg is now ``energy_normalized``.
+  * :py:func:`~rdtools.degradation.degradation_year_on_year`, 
+    :py:func:`~rdtools.degradation.degradation_ols` and 
+    :py:func:`~rdtools.degradation.degradation_classical_decomposition`
+    first kwarg is now ``energy_normalized``.
+  * :py:func:`~rdtools.filtering.normalized_filter` input kwargs are now ``energy_normalized``, ``energy_normalized_low`` and ``energy_normalized_high``.
+  * :py:func:`~rdtools.filtering.poa_filter` input kwargs are now ``poa_global``, ``poa_global_low`` and ``poa_global_high``.
+  * :py:func:`~rdtools.filtering.tcell_filter` input kwargs are now ``temperature_cell``, ``temperature_cell_low`` and ``temperature_cell_high``.
+  * :py:func:`~rdtools.filtering.clip_filter` input kwargs are now ``power_ac`` and ``quantile``.
+  * :py:func:`~rdtools.filtering.csi_filter` first two kwargs are now ``poa_global_measured``, ``poa_global_clearsky``.
+  * :py:func:`~rdtools.normalization.normalize_with_pvwatts` pvwatts_kws dictionary keys have been renamed.
+  * :py:func:`~rdtools.normalization.pvwatts_dc_power` input kwargs are now ``poa_global``, ``power_dc_rated``, ``temperature_cell``, ``poa_global_ref``, ``temperature_cell_ref``, ``gamma_pdc``.
+
+Enhancements
+------------
+* Add new :py:mod:`~rdtools.soiling` module to implement the stochastic rate and
+  recovery method (:pull:`112`).
+* Add new function :py:func:`~rdtools.normalization.normalize_with_expected_power` (:pull:`173`).
+* Add new functions :py:func:`~rdtools.normalization.energy_from_power` and
+  :py:func:`~rdtools.normalization.interpolate` (:pull:`105`, :pull:`108`).
```

Comment:
Should `interpolate_series`, `trapz_aggregate`, and `t_step_nanoseconds`  be mentioned here as well?

Reply:
Perhaps they ought to be made private with leading `_` ?

Reply:
Up to you.  Those three are listed in `api.rst`, so that'll need to be changed if they are made private.